tools/generator/src/Extensions/XCScheme+Extensions.swift

```diff
@@ -83,15 +83,25 @@ extension XCScheme {
 
         let launchAction: XCScheme.LaunchAction
         if let launchActionInfo = schemeInfo.launchActionInfo {
+            let scriptTitle: String
+            let scriptText: String
+            if buildMode == .xcode {
+                scriptTitle = "Update .lldbinit"
+                scriptText = XCScheme.ExecutionAction.createLLDBInitScript
+            } else {
+                scriptTitle = "Update .lldbinit and copy dSYMs"
+                scriptText = XCScheme.ExecutionAction.createLLDBInitScript +
+                    "\n" + XCScheme.ExecutionAction.copyDSYMsScript
+            }
```

Comment:
Why are you adding anything to the Launch action? We are already calling this in the Profile action. Any change to that logic should be its own PR (but I thought we agreed that it shouldn't be the launch action).

Reply:
I wanted to check if this solves [this problem](https://github.com/buildbuddy-io/rules_xcodeproj/issues/1336#issuecomment-1428749738).

Reply:
That seems works